src/main/java/com/coreos/jetcd/Util.java

```diff
@@ -79,4 +84,49 @@ protected static Header apiToClientHeader(ResponseHeader header, long compactRev
     return new Header(header.getClusterId(), header.getMemberId(), header.getRevision(),
         header.getRaftTerm(), compactRevision);
   }
+
+  /**
+   * convert API LeaseTimeToLiveResponse to self defined LeaseTimeToLiveResponse.
+   */
+  static LeaseTimeToLiveResponse apiToLeaseTimeToLiveResponse(
+      com.coreos.jetcd.api.LeaseTimeToLiveResponse response) {
+    List<ByteSequence> byteSequenceKeys = response.getKeysList().stream()
+        .map(byteStringKey -> ByteSequence.fromBytes(byteStringKey.toByteArray()))
+        .collect(Collectors.toList());
+    return new LeaseTimeToLiveResponse(apiToClientHeader(response.getHeader(), 0),
+        response.getID(), response.getTTL(), response.getGrantedTTL(), byteSequenceKeys);
+  }
+
+  /**
+   * convert ListenableFuture of Type S to CompletableFuture of Type T. copied from:
+   * https://github.com/enncloud/jetcd/blob/master/src/main/java/com/coreos/jetcd/EtcdUtil.java
+   */
+  static <S, T> CompletableFuture<T> listenableToCompletableFuture(
```

Comment:
it is not from our branch. it is from enncloud's `https://github.com/enncloud/jetcd/`. So i copied his code.